dts/arm/nxp/nxp_lpc55S6x_common.dtsi

```diff
@@ -314,7 +314,17 @@
 		#io-channel-cells = <1>;
 	};
 
-	usbhs: usbhs@144000 {
+	usbfs: usbfs@84000 {
+		compatible = "nxp,mcux-usbd";
```

Comment:
in the commit message here is says "also fix the usb ram mapping" but that seems to happen in a later commit

Reply:
Thanks for spotting this, I have updated the commit message.